docker/cygnus-ngsi/agent.conf

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of fiware-cygnus (FI-WARE project).
+#
+# fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+# General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+# fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+#
+
+cygnus-ngsi.sources = http-source
+cygnus-ngsi.sinks = mysql-sink
+cygnus-ngsi.channels = mysql-channel
+
+cygnus-ngsi.sources.http-source.channels = mysql-channel
+cygnus-ngsi.sources.http-source.type = org.apache.flume.source.http.HTTPSource
+cygnus-ngsi.sources.http-source.port = 5050
+cygnus-ngsi.sources.http-source.handler = com.telefonica.iot.cygnus.handlers.NGSIRestHandler
+cygnus-ngsi.sources.http-source.handler.notification_target = /notify
+cygnus-ngsi.sources.http-source.handler.default_service = default
+cygnus-ngsi.sources.http-source.handler.default_service_path = /
+cygnus-ngsi.sources.http-source.interceptors = ts gi
+cygnus-ngsi.sources.http-source.interceptors.ts.type = timestamp
+cygnus-ngsi.sources.http-source.interceptors.gi.type = com.telefonica.iot.cygnus.interceptors.NGSIGroupingInterceptor$Builder
+cygnus-ngsi.sources.http-source.interceptors.gi.grouping_rules_conf_file = /opt/apache-flume/conf/grouping_rules.conf
+
+cygnus-ngsi.sinks.mysql-sink.channel = mysql-channel
+cygnus-ngsi.sinks.mysql-sink.type = com.telefonica.iot.cygnus.sinks.NGSIMySQLSink
+cygnus-ngsi.sinks.mysql-sink.enable_grouping = false
+cygnus-ngsi.sinks.mysql-sink.enable_lowercase = false
+cygnus-ngsi.sinks.mysql-sink.mysql_host = mysql
+cygnus-ngsi.sinks.mysql-sink.mysql_port = 3306
+cygnus-ngsi.sinks.mysql-sink.mysql_username =
+cygnus-ngsi.sinks.mysql-sink.mysql_password =
```

Comment:
This is a dummy configuration file, not a real one. It is expected the user provides a real one when using the docker for Cygnus. Even, such a MySQL server does not exist :)